src/main/java/gregtech/client/renderer/texture/custom/QuantumStorageRenderer.java

```diff
@@ -110,20 +138,23 @@ public static void renderChestStack(double x, double y, double z, MetaTileEntity
 
     public static void renderTankFluid(CCRenderState renderState, Matrix4 translation, IVertexOperation[] pipeline,
                                        FluidTank tank, IBlockAccess world, BlockPos pos, EnumFacing frontFacing) {
+        FluidStack stack = tank.getFluid();
+        if (stack == null || stack.amount == 0 || !ConfigHolder.client.enableFancyChestRender)
+            return;
+
         float lastBrightnessX = OpenGlHelper.lastBrightnessX;
         float lastBrightnessY = OpenGlHelper.lastBrightnessY;
+
         if (world != null) {
             renderState.setBrightness(world, pos);
         }
-        FluidStack stack = tank.getFluid();
-        if (stack == null || stack.amount == 0)
-            return;
 
```

Comment:
Why was this removed? It would seem useful to skip rendering if there is no fluid, or the fluid in null.

Reply:
the checks were moved up